Tests/FluentAssertions.Specs/Primitives/DateTimeOffsetAssertionSpecs.cs

```diff
@@ -364,6 +364,189 @@ public void
 
         #endregion
 
+        #region (Not) BeExactly
+
+        [Fact]
+        public void Should_succeed_when_asserting_datetimeoffset_value_is_exactly_equal_to_the_same_value()
```

Comment:
🔧 No need to mention `datetimeoffset` in there since it's clear from the context. Also, we prefer to keep literal names of code elements out of the test name.

Reply:
Gladly. I just copied the Be/NotBe tests, which has the misleading "numeric" name, and chanhed it. Will fix.